nucypher/blockchain/eth/token.py

```diff
@@ -559,6 +575,12 @@ def _do_work(self) -> None:
         if interval > 0:
             self.log.warn(f"MISSED CONFIRMATIONS - {interval} missed staking confirmations detected.")
 
+        # Only perform work this round if the requirements are met
+        if not self.__check_work_requirement():
```

Comment:
Yeah, great!

What is the value of `__check_work_requirement()` when the node just starts, how quickly does it turn `True`?

Reply:
Fast, and assumes 10 at startup.